content/common/makefile/defs.mk

```
@@ -8,5 +8,6 @@ LOGGER_DIR := $(UTILS_DIR)/log
 
 CPPFLAGS += -I$(INCLUDES_DIR)
 CFLAGS += -g -Wall -Wextra
+LDFLAGS = -z lazy
```

```suggestion
LDFLAGS += -z lazy
```
This way the variable set in `process-memory/Makefile` won't be overwritten.